vim/vscode-motion.vim

```diff
@@ -0,0 +1,10 @@
+function! s:toLastCharOfScreenLine()
+    call VSCodeNotify('cursorEnd')
+    " Optmized delay is required after calling VSCode command
```

Comment:
This is a very dirty hack. Have you tried to use `VSCodeCall` instead?

Reply:
I refactor with VSCode commands and removing the `sleep` in https://github.com/asvetliakov/vscode-neovim/pull/455/commits/670718fcddec938acd6b9d43de771621d55f61cf